src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+
```

Comment:
According to the checklist, don't begin class or method implementation with an empty line. Please remove the redundant empty lines above the class declaration.